### PR TITLE
New web-client project a

### PR DESCRIPTION
**Pull request by luyendt**
Create Dashboard view for web-client

### CODE REVIEW

File: web-client/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "web-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.1.1",
+    "font-awesome": "^4.7.0",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
```
Comment (iamacoderguy) [inline web-client/package.json]: Are you sure if using dynamic versions is okay?

File: web-client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (iamacoderguy) [inline web-client/public/index.html]: I think we need a better description of the site. Something relates to the project.

File: web-client/src/components/DashboardView/DashboardView.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          className="result-box"
+          title="Result box"
+          srcDoc="<i>Result...</i>"
+          name="iframe-result"
+        />
+
+        <div style={{ padding: 16 }}>
+          <InputtingForm
+            formMethod="POST"
+            apiUrl="http://localhost:3000/api/files/path?_method=PUT"
```
Comment (iamacoderguy) [inline web-client/src/components/DashboardView/DashboardView.jsx]: It will not work if accessing from outside of the server.

Comment (luyendt) [reply web-client/src/components/DashboardView/DashboardView.jsx]: I create issue #52  to resolve this one

Comment (luyendt) [reply web-client/src/components/DashboardView/DashboardView.jsx]: And issue #51 for writing unit tests

**Pull request merged**